src/OpenLoco/src/Vehicles/Routing.cpp

```diff
@@ -416,8 +416,8 @@ namespace OpenLoco::Vehicles
 
     // 0x004A2AF0
     // Passes occupied state via _routingTransformData
-    // Returns true for signals
-    static bool findSignalsAndOccupation(const LocationOfInterest& interest)
+    // Returns true for signal block end
+    static bool findOccupationByBlock(const LocationOfInterest& interest)
```

Comment:
I've renamed these functions so that its clear that this is the part that limits it by block and the findAll... just walks the whole network.